lib/stdlib/doc/src/gen_server.xml

```diff
@@ -973,7 +984,10 @@ gen_server:abcast     -----> Module:handle_cast/2
           <c>proc_lib:hibernate/3</c></seemfa>).</p>
         <p>If the initialization fails, the function is to return
           <c>{stop,Reason}</c>, where <c>Reason</c> is any term, or
-          <c>ignore</c>.</p>
+          <c>ignore</c>. An exit signal with this reason is sent to linked
+          processes and ports, notably to the process starting the gen_server
+          when <seemfa marker="#start_link/3"><c>start_link/3,4</c></seemfa>
+          is used.</p>
```

Comment:
If `ignore` is returned the `gen_server` process exits with exit reason `normal`.

Reply:
Changing to "`An exit signal with this <c>Reason</c> (or with reason <c>normal</c> if <c>ignore</c> is returned) is sent to`..."